src/ProjectManager.cs

```diff
@@ -157,20 +157,51 @@ public string GetPtusaProjectsPath()
                     StreamWriter sr = new StreamWriter(path, true);
                     sr.WriteLine("[path]\nfolder_path=");
                     sr.Close();
-                    sr.Flush();
+                    sr.Flush();         
                 }
                 PInvoke.IniFile iniFile = new PInvoke.IniFile(path);
 
                 // Считывание и возврат пути каталога проектов
-                string projectsFolder = iniFile.ReadString("path", "folder_path", "");
-                if (projectsFolder.Last() == '\\')
+                string projectsFolders =
+                    iniFile.ReadString("path", "folder_path", "");
+                string projectsFolder = "";
+
+                string[] projectsFoldersArray = projectsFolders.Split(';');
+                string firstPath = "";
+                bool firstPathIsSaved = false;
+                foreach(string pathFromArray in projectsFoldersArray)
                 {
-                    return projectsFolder;
+                    if (pathFromArray != "")
+                    {
+                        projectsFolder = pathFromArray;
+                        if (!firstPathIsSaved)
+                        { 
+                            firstPath = pathFromArray;
+                            firstPathIsSaved = true;
+                        }
+
+                        if (projectsFolder.Last() != '\\')
+                        {
+                            projectsFolder += "\\";
+                        }
+
+                        string folderPath = projectsFolder + projectName;
+                        if (Directory.Exists(folderPath))
+                        {
+                            return projectsFolder;
+                        }
+                    }
+                }
+
+                if(firstPath !="")
+                {
+                    return firstPath;
```

Comment:
```suggestion
                    return firstPath + '\\';
```

Надо добавить разделитель для каталогов, т.к немного некорректно каталоги разделялись.